pepper-apis/config/application.conf.ctmpl

```diff
@@ -2,18 +2,24 @@
 
 {{with $environment := env "ENVIRONMENT"}}
 {{with $version := env "VERSION"}}
+{{with $gae := env "GAE"}}
 {{with $conf := vault (printf "secret/pepper/%s/%s/conf" $environment $version)}}
 {{with $dsm_conf := vault (printf "secret/pepper/%s/%s/dsm/config" $environment $version)}}
 {{with $auth0 := index $conf.Data "auth0"}}
 {{with $testingConf := vault (printf "secret/pepper/%s/%s/testing" $environment $version)}}
 {{with $testingAuth := index $testingConf.Data "auth0"}}
 {{with $slack := index $conf.Data "slack"}}
 {{with $elasticsearchConf := vault (printf "secret/pepper/%s/%s/elasticsearch" $environment $version)}}
-
 {
-    "dbUrl": "{{$conf.Data.dbUrl}}",
+    {{if eq $gae "true"}}
+        "dbUrl": "{{$conf.Data.gaeDbUrl}}",
+        "housekeepingDbUrl": "{{$conf.Data.gaeHousekeepingDbUrl}}",
+    {{else}}
+        "dbUrl": "{{$conf.Data.dbUrl}}",
+        "housekeepingDbUrl": "{{$conf.Data.housekeepingDbUrl}}",
+    {{end}}
+    "defaultTimezone": "{{$conf.Data.defaultTimezone}}",
```

Comment:
This is the main difference in our generated GAE-specific config file: the database URLs are different. Do not use sqlproxy. Permissions need to be set properly.

Reply:
Not sure:
The GAE URLs look like so:
`jdbc:mysql:///pepperdevclone?cloudSqlInstance=broad-ddp-dev:us-central1:ddp-dev&socketFactory=com.google.cloud.sql.mysql.SocketFactory&user=XXXXX&password=XXXXXX&characterEncoding=UTF-8&useLegacyDatetimeCode=false&serverTimezone=UTC&useSSL=false&sessionVariables=innodb_strict_mode=on,tx_isolation='READ-COMMITTED',sql_mode='TRADITIONAL'`
So no reference to `sqlproxy`. I expect we will still need to use sqlproxy when trying to get to databases locally. Perhaps we can figure out a way to update the generated config files to create the sqlproxy urls from the "real" ones using a flag?

Reply:
Hmm, I'm not sure that is necessary? Because for sqlproxy, we just need the sql instance id (which is `broad-ddp-dev:us-central1:ddp-dev` here). Then we just need the user credentials. So the GAE URL has everything we need to connect locally.

Reply:
Right. The GAE URL is the "real" url that sqlproxy is connecting to for us.

If starting from scratch, I would break out the individual fields out of the jdbc url (particularly the credentials) and set those from within the connection creation code. Something distasteful of a URL that includes a password.

But I am trying preserve existing functionality and GAE deployments and our current deployments to live at the same time.

And I am also wary on going on another yak shaving expedition. I wholly agree this can be better but I hope you would agree that we can take on cleaning up the connection URLs as a separate undertaking.